### PR TITLE
Add rule for URI

### PR DESCRIPTION
**Pull request by koppor**

### CODE REVIEW

File: pkgloader-recommended.sty [hunk 1]
```diff
@@ -175,6 +175,25 @@
...
+%  documentation and uses of |\@ifpackageloaded| and |\RequirePackage| in
+%  |uri.sty|:
+%
+%    \begin{macrocode}
+\Load {hyperref} before {uri}
+                 if loaded
+                 because {uri uses hyperref's commands if hyperref is loaded. See also Section 2 of uri.pdf.}
+\Load {kvoptions} before {uri}
+                  if loaded
+                  because {uri has a RequirePackage on kvoptions}
```
Comment (mhelvens) [inline pkgloader-recommended.sty]: I don't think this rule does anything? `uri` already loads `kvoptions`. And `kvoptions` doesn't take any options itself. `url` accepts an option and `uri` doesn't provide any, so I see the reasoning for that one.

File: pkgloader-recommended.sty [hunk 2]
```diff
@@ -175,6 +175,25 @@
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  A list of recommended rules extracted from the uri
+%  documentation and uses of |\@ifpackageloaded| and |\RequirePackage| in
+%  |uri.sty|:
```
Comment (mhelvens) [inline pkgloader-recommended.sty]: Please provide the URL to the documentation. Follow the convention set by the other rules in this file.

(It's http://mirror.ox.ac.uk/sites/ctan.org/macros/latex/contrib/uri/uri.pdf)

Comment (koppor) [reply pkgloader-recommended.sty]: It's http://texdoc.net/pkg/uri -- http://texdoc.net is the new canonical URL scheme for linking documentation.

**Comment by koppor** [comment]
Fixed the issues. 😇

**Pull request merged**